src/views/login/pronote/PronoteInstanceSelector.tsx

```diff
@@ -263,33 +291,39 @@ const PronoteInstanceSelector: Screen<"PronoteInstanceSelector"> = ({
             )}
 
             <Reanimated.View
-              style={[styles.list,
+              style={[
+                styles.list,
                 {
-                  paddingBottom: keyboardHeight + insets.bottom + (keyboardHeight > 0 ? 0 : 20),
-                }
+                  paddingBottom:
+                    keyboardHeight +
+                    insets.bottom +
+                    (keyboardHeight > 0 ? 0 : 20),
+                },
               ]}
-              layout={LinearTransition.springify().mass(1).stiffness(100).damping(40)}
+              layout={LinearTransition.springify()
+                .mass(1)
+                .stiffness(100)
+                .damping(40)}
             >
               {instances.map((instance, index) => (
                 <Reanimated.View
                   style={{ width: "100%" }}
-                  layout={LinearTransition.springify().mass(1).stiffness(150).damping(20)}
+                  layout={LinearTransition.springify()
+                    .mass(1)
+                    .stiffness(150)
+                    .damping(20)}
                   entering={
-                    index < 10 &&
-                    !hasSearched ?
-                      FlipInXDown.springify().delay(100 * index)
+                    index < 10 && !hasSearched
+                      ? FlipInXDown.springify().delay(100 * index)
                       // @ts-expect-error
-                      : ZoomInEasyDown.duration(400).easing(Easing.bezier(0.25, 0.1, 0.25, 1)).delay(30 * index)
+                      :  ZoomInEasyDown.duration(400).easing(Easing.bezier(0.25, 0.1, 0.25, 1)).delay(30 * index)
```

Comment:
Non c'était bien avant :

```suggestion
                      : ZoomInEasyDown.duration(400).easing(Easing.bezier(0.25, 0.1, 0.25, 1)).delay(30 * index)
```

Reply:
En effet, j'avais pas vu 😅